src/main/java/com/microsoft/cse/helium/app/controllers/ActorsController.java

```diff
@@ -43,6 +43,8 @@
       @ApiParam(value = "The ID of the actor to look for", example = "nm0000002", required = true)
       @PathVariable("id")
           String actorId) {
+    logger.info(String.format("getActor (actorId=%s)",actorId));
```

Comment:
I believe I am guilty of this as well, but instead of c-style %d, %s, it is preferable to use the MessageFormat class. e.g. 

    System.out.println(MessageFormat.format("Version                    {0}",
        buildConfig.getBuildVersion()));.

Example is present in DryRunProcessor.java

Reply:
Updated all String.format() that I added to MessageFormat.format().